### PR TITLE
add basic list of review infos

### PR DESCRIPTION
**Pull request by bastelfreak**
Hi,

I would like to add a list of questions a reviewer can go through while checking a pull request. This list isn't complete. I would like to get this to our website as soon as possible and update it from time to time

### CODE REVIEW

**Comment by bastelfreak** [comment]
Dear @voxpupuli/collaborators I would like to get some feedback here.

**Comment by wyardley** [comment]
This looks good to me

**Comment by bastelfreak** [comment]
@wyardley I added:

* Are facts used? They should only be accessed via `$facts[]`, not topscope variables

File: docs/index.md [hunk 1]
```diff
@@ -111,3 +111,15 @@ Run the rake target `travis_release`. This will:
...
 Travis will then kick off a build against the new tag created and deploy that build to the forge. Caution: The Vox Pupuli repo has to be the configured default branch in your local clone. Otherwise you will try to release to your fork.
+
+## Reviewing a module
+
+There are a few things that can be checked if you review a pull request against one of our modules:
+
+* Correct email address used in the commits (if so, travis displays the avatar next to the commit)?
+* Is this a bugfix, modulesync, breaking change, enhancement, docs update? Label it with `bug`, `modulesync`, `backwards-incompatible`, `enhancement`, `docs`
+* Are updates to the README.md needed but missing? Label it with `needs-docs`
+* Are there merge conflicts? Add the `needs-docs` label
```
Comment (alexjfisher) [inline docs/index.md]: Should be `needs-rebase`

File: docs/index.md [hunk 2]
```diff
@@ -111,3 +111,15 @@ Run the rake target `travis_release`. This will:
...
+* Does it need additional tests? Add the `needs-tests` label
+* Are new parameter introduced? They should have datatypes
+* Are facts used? They should only be accessed via `$facts[]`, not topscope variables
```
Comment (alexjfisher) [inline docs/index.md]: or the new stdlib function?

Comment (bastelfreak) [reply docs/index.md]: lol. Is this the workaround for facter[] not working properly with nonexistent nested hashes? :(

File: docs/index.md (hunk 1 above)
Comment (bastelfreak) [reply docs/index.md]: fixed

**Comment by alexjfisher** [comment]
Something about the PR title as it will form the entry in changelog?
Do commits need squashing?  (More than 1 commit is often fine.  But we've also seen cases of tens of commits that just flip syntax back and forth)
Should changes be split into multiple PRs?  If two distinct things are being fixed/implemented, perhaps separate PRs would be better. (I'm thinking of the changelog again.) :)

**Comment by TraGicCode** [comment]
Great stuff guys.  Looking good!

File: docs/index.md [hunk 3]
```diff
@@ -111,3 +111,15 @@ Run the rake target `travis_release`. This will:
...
+## Reviewing a module
+
+There are a few things that can be checked if you review a pull request against one of our modules:
+
+* Correct email address used in the commits (if so, travis displays the avatar next to the commit)?
+* Is this a bugfix, modulesync, breaking change, enhancement, docs update? Label it with `bug`, `modulesync`, `backwards-incompatible`, `enhancement`, `docs`
+* Are updates to the README.md needed but missing? Label it with `needs-docs`
+* Are there merge conflicts? Add the `needs-rebase` label
+* Does it need additional tests? Add the `needs-tests` label
+* Are new parameter introduced? They should have datatypes
```
Comment (alexjfisher) [inline docs/index.md]: parameters with an 's'?

**Comment by bastelfreak** [comment]
I am going to merge this. We should update it from time to time, but it is a good start.

**Pull request merged**